code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -34,8 +34,11 @@
 	healable = 0
 	environment_smash = ENVIRONMENT_SMASH_STRUCTURES
 	obj_damage = 50
-	melee_damage_lower = 30
-	melee_damage_upper = 30
+	melee_damage_lower = 20 // reduced from 30 to 20 with wounds since they get big buffs to slicing wounds
```

Comment:
big, BIG (s)laughter demon nerf

the wound bonus shit isn't enough to compensate for this, and even if it was, a slaughter demon usually needs to CRIT people, not just give them a wound and run off